src/openMVG/matching/cascade_hasher.hpp

```diff
@@ -1,27 +1,7 @@
-// Copyright (c) 2015 Pierre MOULON.
-
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-//------------------
-//-- Bibliography --
-//------------------
-//- [1] "Fast and Accurate Image Matching with Cascade Hashing for 3D Reconstruction"
-//- Authors: Jian Cheng, Cong Leng, Jiaxiang Wu, Hainan Cui, Hanqing Lu.
-//- Date: 2014.
-//- Conference: CVPR.
-//
-// This implementation is based on the Theia library implementation.
-//
-// Update compare to the initial paper [1] and initial author code:
-// - hashing projection is made by using Eigen to use vectorization (Theia)
-// - replace the BoxMuller random number generation by C++ 11 random number generation (OpenMVG)
-// - this implementation can support various descriptor length and internal type (OpenMVG)
-// -  SIFT, SURF, ... all scalar based descriptor
-//
+// This file is part of the AliceVision project and is made available under
+// the terms of the MPL2 license (see the COPYING.md file).
 
-// Copyright (C) 2014 The Regents of the University of California (Regents).
+// Copyright (C) 2013 The Regents of the University of California (Regents).
```

Comment:
Why keep this copyright and not some others?
Does my copyright have no meaning and some other do?

Reply:
You are part of the copyright holders of the whole project.
But these files have been taken from another project and included here with another license. So it the same problem than for "translation_averaging_solver_l2_chordal.cpp", we cannot relicensing it and neither change the copyright.